[Docs Update]: Review and Update iOS Setup documentation (#6727)
mattermost/docs: source/collaborate/install-ios-app.rst

1. On your device, visit the App Store. 
2. Search for "Mattermost" and select **GET** to download the app.
3. Enter your Apple ID Password or use Face or Touch ID to proceed with the installation. You may also be asked to provide an Apple Verification Code received through text/phone call, login notification, or **Settings** on your trusted device.

> 4. Open the Mattermost app and click on **Sign in** on the welcome screen which is below the **Next** button.

Reviewer: ```suggestion
4. Open the Mattermost app and tap **Sign in** below the **Next** button.
```

> 5. On the next screen, click on **Allow** when prompted to approve notifications for the Mattermost app.

Reviewer: ```suggestion
5. On the next screen, tap **Allow** when prompted to approve notifications for the Mattermost app.
```

6. Now follow the steps below to log in:

   a. **Enter Server URL:** This is the web address you go to when you want to access Mattermost. 

>    b. **Display Name:** This is a name for your server so that you can identify it in case you have multiple servers set up in your app. Click on **Connect** to continue.

Reviewer: ```suggestion
   b. **Display Name:** This is a name for your server so that you can identify it in case you have multiple servers set up in your app. Tap **Connect** to continue.
```

>    c. Enter your credentials as specified by your Mattermost system admin in the next screen. Click on **Log in**.

Reviewer: ```suggestion
   c. Enter your credentials as specified by your Mattermost system admin in the next screen. Tap **Log in**.
```
